common/changes/@rushstack/typings-generator/halfnibble-make-typings-async_2020-09-11-02-15.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/typings-generator",
+      "comment": "Change TypingsGenerator to support async methods. Typings generation now returns a promise.",
+      "type": "major"
```

```suggestion
      "type": "minor"
```

Let's not release version `1.0.0` yet.